View3D/Services/MeshOptimizerService.cs

```diff
@@ -103,6 +123,31 @@ public static MeshObject CreatedReducedCopy(MeshObject original, float factor)
             clone.RebuildVertexBuffer();
 
             return clone;
+        }      
+
+        private static MeshObject GetReducedMeshSimplygon(ISimplygon sg, MeshObject originalMesh, float factor)
+        {
+            // -- Copy MeshObject data into Simplygon containers
+            SimplygonHelpers.InitSGGeometryDataObject(sg, out var pGeometryData, originalMesh);
```

Comment:
Consider not putting p in front of the variable. It adds no real value, breaks with the naming standard and makes it less intuitive to type with IntelliSense